pkg/apis/rbac/register.go

```diff
@@ -22,7 +22,7 @@ import (
 	"k8s.io/kubernetes/pkg/watch/versioned"
 )
 
-const GroupName = "rbac.authorization.k8s.io"
+const GroupName = "rbac"
```

Comment:
Copied from @deads2k comment at https://github.com/kubernetes/kubernetes/issues/25860#issuecomment-220587237:

> It nests under the existing `authorization.k8s.io` and matches the `authentication.k8s.io`, which also follows our plan to make the names dns names to avoid any ambiguity.  We most recently talked about it here: https://github.com/kubernetes/kubernetes/pull/24902#discussion_r62391331.